CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to Grafana's Shared Workflows Repository
+
+Welcome to the Shared Workflows repository! This repository contains reusable actions designed to streamline and automate common tasks across multiple departments and external organizations. We appreciate your interest in contributing to our shared efforts.
+
+## Scope of `shared-workflows`
+This repository is for generic workflows and actions which can be used across multiple types of projects. For more specialised use-cases, we encourage the use of project- or area-specific shared workflow repositories.
+Examples which would fit in here:
+- Push a container image to a Docker registry
+- Run security scanners on built artifacts
+- Read and write objects to cloud object storage (GCS or S3)
+  And those which would be better in project- or area-specific shared workflow repositories:
+- Implement a release process specific to a subset of projects
+- Lint project(s) with your team's opinionated toolset
+
+Get in touch with the maintainers [via an issue](https://github.com/grafana/shared-workflows/issues/new) if you're unsure in a particular case.
+
+## Types of Workflows
+
+### 1. Reusable Workflows
+Reusable workflows are located in the `.github/workflows/` directory. These workflows are primarily intended for internal use within this repository. They should be used for automating tasks specific to this repository's needs. Reusable workflows are also required if you intend to use the `on: workflow_call` event for workflow triggering. For more information on reusing workflows and using `on: workflow_call`, refer to the [GitHub documentation](https://docs.github.com/en/actions/using-workflows/reusing-workflows).
+
+### 2. Custom Actions
+Custom actions are defined in the `actions/` directory. These actions are designed for external contributors to add new functionalities and actions that can be used across various projects. If you have innovative ideas or new actions to contribute, the `actions/` directory is the place to do so. To learn more about creating custom actions, check out the [GitHub documentation](https://docs.github.com/en/actions/creating-actions/about-custom-actions).
+
+## Contribution Guidelines
+
+1. **Fork the Repository:** Start by forking the repository to your GitHub account.
+
+2. **Clone the Fork:** Clone your forked repository to your local machine.
+
+3. **Create a Branch:** Create a new branch for your contribution. Use a descriptive name related to the changes you're making.
+
+4. **Make Changes:** Make your desired changes to the codebase.
```

Comment:
Perhaps also something about providing test-cases where possible (keep forgetting that myself all the time)? 🙂

Reply:
Could you give an example?